aspnetcore/aspnetcore-2.1.md

```diff
@@ -0,0 +1,188 @@
+
+For more information, see [ASP.NET Core SignalR](xref:signalr/index)
+
+## Razor class libraries
+
+ASP.NET Core 2.1 makes it easier to build and include Razor-based UI in a library and share it across multiple projects. The new Razor SDK enables building Razor files into a class library project that can be packaged into a NuGet package. Views and pages in libraries are automatically discovered and can be overridden by the application. By integrating Razor compilation into the build:
+
+* The app startup time is significantly faster.
+* Fast updates to  Razor views and pages at runtime are still available as part of an iterative development workflow.
+
+For more information, see [ASP.NET Core Razor SDK](xref:mvc/razor-pages/sdk)
+
+## Identity UI library & scaffolding
+
+### Identity as a library
+
+ASP.NET Core Identity provides a framework for setting up authentication and identity concerns for a website, including user registration, managing passwords, two-factor authentication, social logins and much more.
+
+For ASP.NET Core 2.1 a default identity UI implementation as a library will be provided. The default identity UI can be added to an app by enabling it in `Startup`:
+
+[!code-csharp[Main](aspnetcore-2.1/sample/Startup.cs?name=snippet&highlight=15,39)]
+
+### Identity scaffolder
+
+ASP.NET Core 2.1 and later provides [ASP.NET Core Identity](xref:security/authentication/identity) as a [Razor Class Library](xref:mvc/razor-pages/ui-class). Applications that include Identity can apply the scaffolder to selectively add the source code contained in the Identity Razor Class Library (RCL). You might want to generate source code so you can modify the code and change the behavior. For example, you could instruct the scaffolder to generate the code used in registration. Generated code takes precedence over the same code in the Identity RCL.
+
+Applications that do **not** include authentication can apply the scaffolder to add the RCL Identity package. You have the option of selecting Identity code to be generated.
+
+For more information, see [Scaffold Identity in ASP.NET Core projects](xref:security/authentication/scaffold-identity)
+
+## HTTPS
+
+With the increased focus on security and privacy, enabling HTTPS for web apps is more important than ever before. HTTPS enforcement is becoming increasingly strict on the web. Sites that don’t use HTTPS are considered insecure. Browsers are starting to enforce that many new and existing web features must only be used from a secure context (Chromium, Mozilla). GDPR requires the use of HTTPS to protect user privacy. While using HTTPS in production is critical, using HTTPS during development can also help prevent related issues before deployment, like insecure links. For more information see:
+
+* [HTTP Strict Transport Security Protocol (HSTS)](xref:security/enforcing-ssl#http-strict-transport-security-protocol-hsts)
```

Comment:
These links are already included in the subsections below. How about ending this intro paragraph with something like: "ASP.NET Core 2.1 includes a number of improvements that make it easier to use HTTPS in development and to configure HTTPS in production." Then let the following subsections go into details.